Rivet/Functions/Get-RivetConfig.ps1

```diff
@@ -296,11 +296,20 @@ function Get-RivetConfig
                 {
                     foreach( $dbName in $order )
                     {
+                        if( $dbName -eq "*" )

```

Comment:
I think you can do this and let PowerShell handle the wildcards:

```powershell
foreach ($dbName in $order)
{
    $dbPath = Join-Path -Path $configuration.DatabasesRoot -ChildPath $dbName
    if (-not (Test-Path -Path $dbPath -PathType Container))
    {
        if (-not [wildcardpattern]::ContainsWildcardCharacters($dbName))
        {
            Write-ValidationError  "database named ""$($dbName)"" at ""$($dbPath)"" does not exist"
        }
        continue
    }
    Get-Item -Path $dbPath
}
```
We can also reduce all this nesting in the `Invoke-Command` block:
```powershell
            if( $Database )
            {
                return $Database | 
                    Add-Member -MemberType ScriptProperty -Name Name -Value { $this } -PassThru |
                    Add-Member -MemberType ScriptProperty -Name FullName -Value { Join-Path -Path $configuration.DatabasesRoot -ChildPath $this.Name } -PassThru
            }
            

            if (-not $order)
            {
                return Get-ChildItem -Path $configuration.DatabasesRoot -Directory
            }
            
            # foreach loop here
            foreach( $dbName in $order )
```